README.md

```diff
@@ -595,6 +595,32 @@ Untracked files:
 nothing added to commit but untracked files present (use "git add" to track)
 ```
 
+### masterAdm
```

Comment:
Because I think there will be other features that will be added to this single master admin command which will be selectable (and perhaps combined with) form a list of features.